velox/expression/SignatureBinder.cpp

```diff
@@ -24,6 +24,59 @@ namespace {
 bool isAny(const TypeSignature& typeSignature) {
   return typeSignature.baseType() == "any";
 }
+
+bool isNumber(const std::string& str) {
```

Comment:
Looks like this is an improvement to the existing logic. Would you describe it in the PR description?